sklearn/preprocessing/data.py

```diff
@@ -124,7 +124,7 @@ def scale(X, axis=0, with_mean=True, with_std=True, copy=True):
     scaling using the ``Transformer`` API (e.g. as part of a preprocessing
     :class:`sklearn.pipeline.Pipeline`)
     """
-    X = check_array(X, accept_sparse='csr', copy=copy, ensure_2d=False,
+    X = check_array(X, accept_sparse='csc', copy=copy, ensure_2d=False,
```

Comment:
How much faster is converting to csc and multipying compared to not converting? ``['csc', 'csr']`` will not convert to CSR but will let CSR pass.